controllers/metrics/horizontalrunnerautoscaler.go

```diff
@@ -0,0 +1,67 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/summerwind/actions-runner-controller/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	hraName      = "horizontalrunnerautoscaler"
+	hraNamespace = "namespace"
+)
+
+var (
+	horizontalRunnerAutoscalerMetrics = []prometheus.Collector{
+		horizontalRunnerAutoscalerMinReplicas,
+		horizontalRunnerAutoscalerMaxReplicas,
+		horizontalRunnerAutoscalerDesiredReplicas,
+	}
+)
+
+var (
+	horizontalRunnerAutoscalerMinReplicas = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "horizontal_runner_autoscaler_min_replicas",
```

Comment:
Sorry for the back-and-forth but according to how [kube-state-metrics using `horizontalpodautocaler` instead of `horizontal_pod_autoscaler`](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/horizontalpodautoscaler-metrics.md), this could:

```suggestion
			Name: "horizontalrunnerautoscaler_min_replicas",
```

Reply:
Exactly. I changed the name to `horizontalrunnerautoscaler_spec_min_replicas` like kube-state-metrics.